src/aws-elasticache/user-group.ts

```diff
@@ -0,0 +1,209 @@
+import { ArnFormat, IResource, Lazy, Names, Resource, Stack, Token, aws_elasticache } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { IUser } from './user';
+import { Engine } from './util';
+
+/**
+ * A User Group
```

Comment:
Let's keep it in line with IIamUser and others, just in case.

```suggestion
 * Interface for a User Group
```